### ISSUE TITLE
Should Unrolled test display data table row in HTML report?

### ISSUE THREAD

**Issue by kwangomango** [renatoathaydes/spock-reports#85]
Non unrolled test displays entire data table in 'Examples:' Is the individual data table row expected to be output for an unrolled test?

edit: i've noticed it does display the particular row when using the md template

**Comment by renatoathaydes** [comment]
Unrolled test iterations are shown individually in a table in the HTML report.

Non-unrolled tests cannot because they are run as a single test. Hope this answers your question.

**Comment by kwangomango** [comment]
Ah, what i mean is each unrolled test displaying the individual table row of data for that iteration. I have attached a screenshot to explain. The unrolled test displays both rows of my data table as expected. I was expecting each unrolled test to display the individual data row used by that test.
![unrolled](https://cloud.githubusercontent.com/assets/11312077/19689103/28115dcc-9ac4-11e6-84bc-8c61217728ad.jpg)

Thanks.

**Comment by renatoathaydes** [comment]
Ah, I see... in the last two tests runs above, you wanted to see a row like this:

```
simple test unrolled for a=1 and b=1
a=1 | b=1 | c = 2
Expect: a+b==c
Where: ---
```

That's a good idea. Even though, currently, you could already use the test description to show all values, it may not be adequate sometimes.

I will try to work on this feature for the next release... but if you feel like you can implement it, feel free to try!

**Comment by kwangomango** [comment]
Thanks. I wasn't sure if this was expected behaviour or not because the template report does write the individual rows:
![templatereport](https://cloud.githubusercontent.com/assets/11312077/19691334/cc173110-9acb-11e6-91e0-d0407e4881af.jpg)

**Comment by rkrdpm** [comment]
@renatoathaydes I will implement this if you have not started on it yet. Please let me know.

**Comment by renatoathaydes** [comment]
feel free to checkout `master` and submit a PR!

Would be nice to do it in both the default HTML report creator and in the MD template report creator which ships with spock-reports. But if you do the HTML report, I can do the MD later.

**Comment by renatoathaydes** [comment]
@rkrdpm correcting myself, please submit a PR on the `next` branch, actually... I use that to keep track of releases.

See https://github.com/renatoathaydes/spock-reports#submitting-pull-requests

**Comment by renatoathaydes** [comment]
This has been fixed by @rkrdpm. Are you participating in Hacktoberfest?

**Comment by renatoathaydes** [comment]
Feature released in version 1.4.0. Thanks!
